orc8r/cloud/go/services/analytics/collector_servicer.go

```diff
@@ -18,32 +18,127 @@ import (
 	"magma/orc8r/cloud/go/services/analytics/calculations"
 	"magma/orc8r/cloud/go/services/analytics/protos"
 	"magma/orc8r/cloud/go/services/analytics/query_api"
+	"magma/orc8r/lib/go/metrics"
 
 	"github.com/golang/glog"
 )
 
 //CollectorService implements the operations of collecting the metrics from CWF service
 type CollectorService struct {
-	calculations  []calculations.Calculation
-	promAPIClient query_api.PrometheusAPI
+	calculations    []calculations.Calculation
+	promAPIClient   query_api.PrometheusAPI
+	userStateMgr    *calculations.UserStateMgr
+	analyticsConfig *calculations.AnalyticsConfig
 }
 
 //NewCollectorService constructs new collector service
-func NewCollectorService(promAPIClient query_api.PrometheusAPI, calculations []calculations.Calculation) *CollectorService {
-	return &CollectorService{promAPIClient: promAPIClient, calculations: calculations}
+func NewCollectorService(analyticsConfig *calculations.AnalyticsConfig, promAPIClient query_api.PrometheusAPI, calculations []calculations.Calculation,
+	userStateMgr *calculations.UserStateMgr) *CollectorService {
+	return &CollectorService{promAPIClient: promAPIClient, calculations: calculations, analyticsConfig: analyticsConfig, userStateMgr: userStateMgr}
+}
+
+// filterResults filters the results to ensure that the user aggregate metrics get filtered out in
```

Comment:
Doc comments should be max 79 characters in length (use rulers in your editor). Also, prefer full punctuation (finish your doc comments with periods -- especially for top-level comments like functions, interfaces, etc)

Reply:
Is this 79 characters still relevant ? I can understand if we are coding in vim. If we are coding in vscode and other editors I don't see a reason to adhere to 79 character rule. Anyways if this is a convention in magma code, I will change this accordingly.